server/app/repository/AccountRepository.java

```diff
@@ -423,4 +425,16 @@ public void updateSerializedIdTokens(
     idTokens.storeIdToken(sessionId, oidcProfile.getIdTokenString());
     account.save();
   }
+
+  /**
+   * Stores applicant id in user profile.
+   *
+   * <p>This allows us to know the applicant id instead of having to specify it in the URL path,
+   * or looking up the account each time and finding the corresponding applicant id.
+   */
+  public void populateApplicantIdInProfile(AccountModel account, CiviFormProfileData profileData) {
+    // Accounts correspond to a single applicant.
+    Long applicantId = account.ownedApplicantIds().stream().findAny().orElseThrow();
+    profileData.addAttribute(APPLICANT_ID_ATTRIBUTE_NAME, applicantId);
+  }
```

Comment:
issue (blocking): repository classes are for database interaction, maybe `CiviFormProfileData` is a good place for this method?

Reply:
Done